### PR TITLE
Resolve JBUGs in const.bal

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
Related to #550

### CODE REVIEW

File: compiler/modules/err/err.bal [hunk 1]
```diff
@@ -5,7 +5,7 @@ public type Message string|Template;
 public type Template object {
     *object:RawTemplate;
     public (readonly & string[]) strings;
-    public (string|int)[] insertions;
+    public (string|int|float)[] insertions;
```
Comment (jclark) [inline compiler/modules/err/err.bal:8]: Why is this changed?

**Review by jclark — CHANGES_REQUESTED** [review]

File: compiler/modules/err/err.bal (hunk 1 above)
Comment (KavinduZoysa) [reply compiler/modules/err/err.bal:8]: Because, at https://github.com/ballerina-platform/nballerina/blob/64b296e57ffe1078106fc4f38109507aeaa87f73/compiler/modules/front/const.bal#L423, type of`value` is float and float is not available in this union `(string|int)[] insertions`.

**Review by jclark — APPROVED** [review]
